README.md

```diff
@@ -1,27 +1,68 @@
-# A10 ACOS CLI Plugin Modules
+## Introduction
+Thunder® ADCs (Application Delivery Controllers) are high-performance solutions to accelerate and optimize critical applications to ensure delivery and reliability.
 
-## Installation 
+A10 Ansible ACOS CLI modules is a custom plugin to do configurations on Thunder via CLI commands. It includes example playbooks to apply on hardware and virtual appliances.
 
+We only support Ansible Suite version >=2.9
 
-We only support Ansible version >=2.9
+## Support Matrix
+
+| ACOS CLI Version | Ansible Version | Ansible Tower | Ansible Automation Platform |Ansible Netcomm |
+| :--------: | :-------: | :-------:  |:-------:  |:-------:  |
+| `1.2.1` |  `2.9.x`, `2.13.x`, `2.14.x`, `2.15.x`, `2.16.x` | `17.1.x` |  Not Support | `>=2.0.0,<3.0.0` |
+| `2.0.0` |  `2.9.x`, `2.13.x`, `2.14.x`, `2.15.x`, `2.16.x` | `17.1.x` |  `2.4.4` | `>=3.0.0,=<6.0.0` |
+
+## How it works
+   1. Install Ansible on your local OS, Please refer below sections for more details.
+   2. Search required Ansible configuration from examples. In case not found create a new one, Please refer below sections for more details.
+   3. Execute Ansible playbooks to apply thunder configuration, Please refer below sections for more details.
+   4. Verify thunder configuration after ansible playbook is applied, Please refer below sections for more details.
+
```

Comment:
This line is repetitive in all 4 points, "Please refer below sections for more details.", please remove from all 4 steps and move it to the bottom after 4th step like this:

> 1. Install Ansible on your local OS.
> 2. Search required Ansible configuration from examples. In case the required example is not found, create a new one.
> 3. Execute Ansible playbooks to apply thunder configuration.
> 4. Verify thunder configuration after ansible playbook is applied.
> 
> Please refer to sections below for more details.